docs/ops/indexing.rst

```diff
@@ -167,160 +182,76 @@ Everything is now ready, and we can use the **datacube** tool to add one or more
 datasets into our Cube by running::
 
 
-    datacube dataset add --auto-match <path-to-dataset-document-yaml>
+    datacube dataset add <path-to-dataset-document-yaml>
 
+Note that this path can be a URI, such as the path to a document on S3.
 
 
 .. _sample-eo-data:
 
-Sample Earth Observation Data
------------------------------
-
-The U.S. Geological Survey provides many freely available, Analysis Ready,
-earth observation data products. The following are a good place to start
-looking.
-
-* Landsat
-    * `USGS Landsat Surface Reflectance - LEDAPS and LaSRC available via ESPA 30m`__
-* MODIS
-    * `MCD43A1 - BRDF-Albedo Model Parameters 16-Day L3 Global 500m`__
-    * `MCD43A2 - BRDF-Albedo Quality 16-Day L3 Global 500m`__
-    * `MCD43A3 - Albedo 16-Day L3 Global 500m`__
-    * `MCD43A4 - Nadir BRDF-Adjusted Reflectance 16-Day L3 Global 500m`__
-
-__ https://espa.cr.usgs.gov/
-__ https://lpdaac.usgs.gov/products/mcd43a1v006/
-__ https://lpdaac.usgs.gov/products/mcd43a2v006/
-__ https://lpdaac.usgs.gov/products/mcd43a3v006/
-__ https://lpdaac.usgs.gov/products/mcd43a4v006/
-
-Once you have downloaded some data, it will need :ref:`metadata preparation
-<prepare-scripts>` before use in the Data Cube.
-
 
 Indexing Data on Amazon(AWS S3)
 ===============================
 
+<<<<<<< HEAD
```

Comment:
hmm... we need to address this conflict here

Reply:
Oops, missed one!